mopet/mopet.py

```diff
@@ -339,8 +336,6 @@ def _store_result(self, result_id, ray_object, run_params):
         :type result_id: int
         :param ray_object: ray object
         :type ray_object: ray object
-        :param run_param: explored parameters of the run
-        :type run_param: dict
```

Comment:
This argument is actually called `run_params`, is there a reason why you delete4d it here?

Reply:
Oh, good point, I overlooked it. I added it again!